CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 5.9.0 (under development)
```

Comment:
isn't `5.9.0` still "under development"?

Reply:
I only plan on making this one commit for the next release. I was planning on tagging the merged commit. But I just realized I'd need to update the setup.py as well. I'll revert this line. Thanks!